operator_ui/src/actions.ts

```diff
@@ -1,17 +1,30 @@
-import * as api from 'api'
-import { AuthenticationError } from './api/errors'
+import { Dispatch } from 'redux'
 import { pascalCase } from 'change-case'
 import normalize from 'json-api-normalizer'
-
-const createAction = type => ({ type: type })
-
-const createErrorAction = (error, type) => ({
+import * as api from './api'
+import {
+  AuthenticationError,
+  BadRequestError,
+  ServerError,
+  UnknownResponseError
+} from './api/errors'
+
+type Errors =
+  | AuthenticationError
+  | BadRequestError
+  | ServerError
+  | UnknownResponseError
+
+const createAction = (type: string) => ({ type: type })
+
+const createErrorAction = (error: Error, type: string) => ({
   type: type,
-  error: error.stack,
-  networkError: true
+  error: error.stack
 })
 
-const curryErrorHandler = (dispatch, type) => error => {
+const curryErrorHandler = (dispatch: Dispatch, type: string) => (
```

Comment:
I know this was already in here, just want to say `curryErrorHandler` is pretty awesome.